book/scale_nodes.rst

```diff
@@ -0,0 +1,356 @@
+
+bundles:
+  ...
+  - Snc\RedisBundle\SncRedisBundle
+  ...
+
+4. **Configure** SncRedisBundle - see 
+   `the documentation <https://github.com/snc/SncRedisBundle/blob/master/Resources/doc/index.md>`_
+
+**app/config/config.yaml:**
+   
+.. code-block:: yaml
+
+   snc_redis:
+       clients:  # configure phpredis client
+           default:
+               type: phpredis
+               alias: default
+               dsn: redis://localhost/1
+           doctrine:
+               type: phpredis
+               alias: doctrine
+               dsn: redis://localhost/2
+
+       doctrine: # use Redis caching for Doctrine 
+           metadata_cache:
+               client: doctrine
+               entity_manager: default
+               document_manager: default
+           result_cache:
+               client: doctrine
+               entity_manager: [default]
+           query_cache:
+               client: doctrine
+               entity_manager: default
+
+5. **Sessions storage**. By default the the parameters.yaml is configured to store the session in files 
+   (*session_handler: *session.handler.native_file*). The simplest way to deal with the sessions in a web farm 
+   configuration is to store them in the DB - just change the value to "*session.handler.pdo*". 
+
+   You can also store them in Redis:
+
+**app/config/config.yaml:**
+   
+.. code-block:: yaml
+
+   snc_redis:
+       clients:
+           ...
+           session:
+               type: phpredis
+               alias: session
+               dsn: redis://localhost/3
+       session:  # configure sessions
+           client: session
+           prefix: session
+           use_as_default: true
+
+6. There are two abstract services used in the Oro Platform that are responsible for the cache saving.
+   *oro.file_cache.abstract*: should be used to cache the data private for each node in a web farm
+   *oro.cache.abstract*: should be used to cache data the data to be shared between the nodes.
+   
+   In our case the oro.cache.abstract* is applicable.
+
+**app/config/config.yaml:**   
+
+.. code-block:: yaml
+   
+   services:
+       oro.cache.abstract:
+           class: Snc\RedisBundle\Doctrine\Cache\RedisCache
+           calls:
+               - [setRedis, [@snc_redis.default]]
+
+
+7. **Annotations** cache configuration:
+
+**app/config/config.yaml:**
+
+.. code-block:: yaml
+
+   framework:
+       annotations:
+           cache: oro.cache.abstract
+
+8. **Doctrine** cache configuration:
+
+**app/config/config.yaml:**
+
+.. code-block:: yaml
+
+   doctrine:
+       orm:
+           query_cache_driver: redis
+           result_cache_driver: redis
+           #metadata_cache_driver does NOT require any changes because it is already a child of oro.cache.abstract
+           #metadata_cache_driver:
+           #            type: service
+           #            id: doctrine.metadata.cache
+           #
+           #src/Oro/Bundle/PlatformBundle/Resources/config/doctrine.yaml
```

Comment:
the doc is related to the last master 
as for 1.8.2 you are right - the definition of "doctrine.metadata.cache" is located in config.yml https://github.com/orocrm/platform-application/blob/1.8.2/app/config/config.yml#L153